examples/symlink_example/do_test

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+
+# Use the Bazel binary specified by the integration test. Otherise, fall back 
+# to bazel.
+bazel="${BIT_BAZEL_BINARY:-bazel}"
+
+# GH249: Do not update build files until the Gazelle extension supports the
+# shape of this example.
+# # Generate Swift external deps and update build files
+# bazel run //:tidy
```

Comment:
Is this true for this example? In other words, does //:tidy work for this example?

Reply:
It seems to work fine. I had copied the `resources_example` as a starting point for this example.

I pushed a commit to run `//:tidy` as part of `do_test`